src/modules/nats/nats-container.ts

```diff
@@ -93,7 +92,42 @@ export class StartedNatsContainer extends AbstractStartedContainer {
     };
   }
 
-  public getConnectionOptions(): ConnectionOptions {
+  public getConnectionOptions(): NatsConnectionOptions {
     return this.connectionOptions;
   }
 }
+
+export interface NatsConnectionOptions {
```

Comment:
You will need to export these types via the index.ts 
(so for example you can do `import {NatsContainer} from 'testcontainers'`